question-service/controller/question-controller.js

```diff
@@ -1,4 +1,5 @@
-import Question from '../model/question-model.js';
+import {Question, QuestionCategory} from '../model/question-model.js';
+import {findQuestionById, createQuestion} from '../model/repository.js'
 
 // Get all questions
 export const getAllQuestions = async (req, res) => {
```

Comment:
Since right now we are letting `repository.js` to handle the requests, 
I think we can also move `await Question.find()` to `repository.js` and create an abstraction here.